### PR TITLE
Video Block #39

### PR DESCRIPTION
**Pull request by arumsey**
Fix [#39](https://github.com/hlxsites/merative/issues/39)

##  🔗 : Test URLs
- Before: https://main--merative2--hlxsites.hlx.page/drafts/arumsey/video-block
- After: https://39-video-block--merative2--hlxsites.hlx.page/drafts/arumsey/video-block

## 🗒️ Description

- supports embedded video
- supports external vimeo video
- dark/purple modes

image

### CODE REVIEW

File: blocks/leadspace/leadspace.css [hunk 1]
```diff
@@ -299,66 +299,6 @@ main div.leadspace-wrapper {
     display: none;
 }
 
-/* Video modal */
```
Comment (arumsey) [inline blocks/leadspace/leadspace.css:302]: Moving to `styles.css` to be shared with `leadspace` and `video` blocks.

File: blocks/video/video.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  } else {
+    link.setAttribute('target', '_blank');
+    link.classList.add('open-video');
+  }
+
+  playBtn.innerHTML = `<span>${label}</span>`;
+  return playBtn;
+};
+
+const buildVideoModal = (href) => {
```
[the remark this answers is not in the document]
Comment (amol-anand) [reply blocks/video/video.js:32]: if already being reused, worth putting into scripts.js in a separate story later on.

**Comment by amol-anand** [comment]
The last (3rd video) on the test page doesn't work. is that something to do with the content ?

**Review by amol-anand — APPROVED** [review]

**Comment by arumsey** [comment]
> The last (3rd video) on the test page doesn't work. is that something to do with the content ?

I am working on adding YouTube support which is what the last (3rd) video points to.

**Pull request merged**